evaluation.py

```diff
@@ -156,7 +168,9 @@ def visualize_keypoints(self, X_batch, keypoints_batch, img_id_batch, show_skele
 
     def heatmaps_to_keypoints_batch(self, heatmaps_batch, threshold=HM_TO_KP_THRESHOLD):
         keypoints_batch = []
-        for i in range(len(heatmaps_batch)):
+
+        # dimensions are (num_hg_blocks, batch, x, y, keypoint)
+        for i in range(heatmaps_batch.shape[1]):
```

Comment:
Good catch, was this affecting anything?

Reply:
Yeah, it briefly broke the stacked heatmap creation. I think I introduced it in my last PR by accident.